### PR TITLE
Issue fix: Bruce 4414: Get Individual task With X-Auth should give task details

### PR DESCRIPTION
**Pull request by lakkundipooja**
This PR contains code changes in Del() function in redis.go file

### CODE REVIEW

File: svc-aggregation/agmodel/system.go [hunk 1]
```diff
@@ -316,21 +316,21 @@ func deletefilteredkeys(key string) error {
 	}
 	for _, value := range sf.SearchKeys {
 		for k := range value {
-			delErr := conn.Del(k, key)
+			delErr := conn.Del(k, 0, "*"+key+"*")
```
Comment (amar-shalgar) [inline svc-aggregation/agmodel/system.go]: Please check for all the possibilities of key deletion using *

File: lib-persistence-manager/persistencemgr/redis.go [hunk 2]
```diff
@@ -939,41 +939,53 @@ func (p *ConnPool) GetTaskList(index string, min, max int) ([]string, error) {
 1. index is the name of the index under which the key needs to be deleted
 2. key is the id of the resource to be deleted under an index
 */
-func (p *ConnPool) Del(index, key string) error {
+func (p *ConnPool) Del(index string, cursor float64, key string) error {
```
Comment (bharath-b-hpe) [inline lib-persistence-manager/persistencemgr/redis.go]: why cursor value is required to be passed by the caller?

File: svc-aggregation/agmodel/system.go (hunk 1 above)
Comment (lakkundipooja) [reply svc-aggregation/agmodel/system.go]: addressed

File: lib-persistence-manager/persistencemgr/redis.go (hunk 2 above)
Comment (lakkundipooja) [reply lib-persistence-manager/persistencemgr/redis.go]: addressed

**Review by bharath-b-hpe — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]
LGTM

**Review by skhan-anjum — APPROVED** [review]

**Review by a-ajith — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**